spec/card_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
+require 'support/factory_girl.rb'
 
 describe Card do
   before do
-    @card = Card.create(original_text: "How are you?", translated_text: "Kamusta?", review_date: DateTime.now)
+    @card = create(:card)
```

Comment:
@kennyfrc let – is lazy, let! is not
normal initialization are not using in the test (in general)